apps/dotcom/client/src/tla/components/TlaErrorContent/TlaErrorContent.tsx

```diff
@@ -46,7 +44,7 @@ export function TlaErrorContent({ error }: { error: TlaPageErrorType }) {
 			return (
 				<div className={styles.container}>
 					<p className="tla-text_ui__regular">
-						{raw('Sorry, you don’t have access to that workspace.')}
```

Comment:
When does this error occur?

Reply:
I'm not exactly sure, but I think David stubbed it out for future groups?